ontobio/io/qc.py

```diff
@@ -511,7 +510,8 @@ def test(self, annotation: association.GoAssociation, config: assocparser.AssocP
     "GoRule42": GoRule42(),
     "GoRule43": GoRule43(),
     "GoRule46": GoRule46(),
-    "GoRule50": GoRule50()
+    "GoRule50": GoRule50(),
+    "GoRule13": GoRule13()
```

Comment:
I'd add a comment here for the rationale, so that somebody doesn't switch it back; e.g.:
```python
// WARNING: This rule appears last as we want to clean up the input as much as possible before invoking a rule that depends on an external program. See issue XYZ for some history.
```

Reply:
Ah, I see it up there at the top now. Honestly, would you look at the top before switching it back?

Reply:
I put it at the top since it's sort of a python idiom to document classes and functions from the top.